core/config/config.go

```diff
@@ -86,8 +92,8 @@ func Load(ctx context.Context, db pg.DB, rDB *raft.Service) (*Config, error) {
 				// If we got this far but failed to delete from PG, it's really NBD. Just
 				// log the failure and carry on.
 				log.Error(ctx, err, "failed to delete config from postgres")
-
 			}
+			err = migrateAccessTokens(ctx, db, rDB)
```

Comment:
Alright, I added a panic here - do you think we need to do anything else? (When you say, "get the customer to call us," you mean that people will call if we crash, not that we should print a sentence here like, "Please call us." Right?)